department-of-veterans-affairs/va.gov-cms
# VACMS-5437: Enable pathauto based on menu item placement for VAMC detail pages.

## Description
See #5437 & #4773

## Testing done
Visual / behat

## QA steps
As an administrator, visit `/node/1001/edit` and visually verify that the `Generate automatic URL alias` box is checked

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
